tests/DevApps/WebAppCallsMicrosoftGraph/appsettings.json

```diff
@@ -20,8 +20,7 @@
             {
                 "SourceType": "KeyVault",
                 "KeyVaultUrl": "https://webappsapistests.vault.azure.net",
-                "KeyVaultCertificateName": "Self-Signed-5-5-22",
-                "ManagedIdentityClientId": "9c5896db-a74a-4b1a-a259-74c5080a3a6a"
```

Comment:
why remove it?

Reply:
This was the wrong appsettings, the change was supposed to be in the webappcallswebapicallsgraph appsettings this one is just webappcallsmicrosoftgraph